edx_event_bus_kafka/internal/tests/test_producer.py

```diff
@@ -137,3 +139,39 @@ def test_send_to_event_bus(self, mock_get_serializers):
             on_delivery=ep.on_event_deliver,
             headers={'ce_type': 'org.openedx.learning.auth.session.login.completed.v1'},
         )
+
+    def test_polling_loop(self):
```

Comment:
Is there a way to test errors and happy path separately? Having them both here at once makes it more difficult to understand the test, and might get us in trouble if we want to start having different behavior on error as part of sad-path work.

Reply:
Hmm... yeah, I think I can do that.